flutter_local_notifications/lib/src/platform_specifics/android/notification_details.dart

```diff
@@ -237,6 +238,14 @@ class AndroidNotificationDetails {
   /// Specifies if the notification should automatically dismissed upon tapping
   /// on it.
   final bool autoCancel;
+  
+  /// Specifies if the notification is selectable.
+  /// 
+  /// If `selectable` is `false`, the `contentIntent` will be set to `null`, 
+  /// disabling click actions on the notification.
+  /// 
+  /// The default value is `true`.
+  final bool selectable;
```

Comment:
- Can you update the example app to include a scenario that makes use of this?
- I assume the `onSelectNotification` callback won't fire. Can you confirm? If so, can you update the API docs to explicitly add mention of this. Some developers may not realise the implication based on reading what's there